api/v1alpha1/dspipeline_types.go

```diff
@@ -42,9 +42,6 @@ type DSPASpec struct {
 	*ObjectStorage `json:"objectStorage"`
 	*MLMD          `json:"mlmd,omitempty"`
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default:={deploy: false}
```

Comment:
no deprecation period first?

Reply:
this was never part of any release, so we don't need this